addons/api/fnc_addRackToVehicle.sqf

```diff
@@ -15,18 +16,19 @@
  *   7: Components <ARRAY> (default: [])
  *   8: Connected intercoms <ARRAY> (default: [])
  * 2: Force initialisation <BOOL> (default: false)
+ * 3: Condition called with argument "_unit" <STRING> (default: "")
```

Comment:
Just take code instead of string, no need to `compile`. Should also be advised that it's a pre-compiled function if longer.